test/manual-test-examples/webgl/pixels/sketch.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * webgl loadPixels() and get() example
+ */
+
+var c;
+
+function setup() {
+  createCanvas(500,500,WEBGL);
+}
+
+function draw() {
+  pointLight(200,200,200,0,0,0);
+  background(255, 0, 100);
+  translate(150, 100, 0);
+  push();
+  rotateZ(frameCount * 0.02);
+  rotateX(frameCount * 0.02);
+  rotateY(frameCount * 0.02);
+  fill(0,0,250);
+  if(c)
+  {
+    texture(c);
+  }
+  box(75);
+  pop();
+}
+
+function mousePressed() {
+  c = get();
+}
```

Comment:
can we add a few unit tests that aren't manual, too? these automated unit tests are the ones that get run with every grunt build so it's easier to catch things that break with them.